### PR TITLE
Correct number of syllables on first line of haiku

### PR DESCRIPTION
**Pull request by deepyaman**
No matter how I read it, I couldn't reconcile the original first line to 5 syllables.

An alternative fix would be `SARS Covid19`, but the proposed correction is more common when starting with `SARS`.

### CODE REVIEW

File: _haikus/please_go_away_covid.md [hunk 1]
```diff
@@ -4,6 +4,6 @@ title: Please go away Covid
 author: Amogh Rajesh Desai
 ---
 
-SARS Covid19 Virus<br>
+SARS-CoV-2 virus<br>
```
Comment (deepyaman) [inline _haikus/please_go_away_covid.md:7]:
```suggestion
SARS-CoV-two virus<br>
```

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
Thank you for your interest in Cloud Haiku! Please submit one new, original haiku in markdown format in order to contribute. Edits on existing haikus do not count as a contribution. Please consult our [contributing guidelines](https://github.com/do-community/cloud_haiku#contributing) for more guidance on how to create a valid pull request.

*Please note: after more than two days of inactivity, pull requests are subject to being closed and marked invalid by maintainers.*

**Comment by deepyaman** [comment]
@ltagliaferri Thank you for your comment. I'm not interested in this from a contribution count perspective; I already have more than enough PRs for Hacktoberfest:

image

I want to fix this because I don't believe it is a valid haiku. Should I just ignore this and try fixing number handling in https://github.com/words/syllable for the future instead?

**Comment by ltagliaferri** [comment]
Sure, working on a syllable counter sounds good to me! The reason why we don't want others to edit others' file doesn't have to do with Hacktoberfest, but that we want each author to have control over the text. Thank you.

**Pull request closed without merging**